packages/typespec-ts/test/modularIntegration/lroModular.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { StandardClient } from "../modularIntegration/generated/lro_modular/generated/src/index.js";
+import { assert } from "chai";
+import {
+  StandardContext,
+  beginCreateOrReplace,
+  beginDelete,
+  beginExport,
+  createStandard
+} from "./generated/lro_modular/generated/src/api/index.js";
+describe("LRO StandardClient Classical Client", () => {
+  let client: StandardClient;
+
+  beforeEach(() => {
+    client = new StandardClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  it("should put LRO response", async () => {
```

Comment:
Is there a case, where service has specified final-state-via ?

Reply:
Not sure we have this case in DPG now, in this pr we only support the 3 standard cases in cadl-ranch.